flake.nix

```diff
@@ -8,14 +8,20 @@
   inputs.nix.inputs.nixpkgs.follows = "nixpkgs";
   inputs.nix.inputs.libgit2.follows = "libgit2";
 
+  inputs.nix-eval-jobs.url = "github:nix-community/nix-eval-jobs";
+  inputs.nix-eval-jobs.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
Something I forgot in the last review: I think we should also force the nix version of hydra into the package, otherwise we might depend on 2 nix versions

Reply:
I can maintain release branches for different nix versions again if this is needed.

Reply:
I don't think the literal Nix derivation is forced to be the same, but the version is the same now.